jvb/src/main/kotlin/org/jitsi/videobridge/octo/OctoRelayService.kt

```diff
@@ -48,8 +48,8 @@ class OctoRelayService {
             throw IllegalStateException("Octo relay service is not enabled")
         }
 
-        val address = config.bindAddress
-        val port = config.bindPort
+        val address = config.bindAddress ?: throw IllegalStateException("bind-address not configured.")
```

Comment:
In the case where `bindAddress` isn't set but `enabled` is true (i.e. you only want to use OctoV2) this will throw `IllegalStateException`, which will cause the `Error initializing OctoRelayService` to be logged, which might be confusing?